src/main/external-resources/renderers/BlackBerry-PlayBook-KalemSoftMP.conf

```diff
@@ -39,7 +39,7 @@ Supported = f:wmv                 m:video/x-ms-wmv
 Supported = f:wav    m:audio/wav        n:6   a:dts|lpcm     s:48000
 Supported = f:wav    m:audio/wav        n:2                  s:48000
 Supported = f:mp3    m:audio/mpeg       n:2
-Supported = f:aac    m:audio/x-m4a      n:2   a:(?!alac).+
+Supported = f:m4a    m:audio/x-m4a      n:2   a:(?!alac).+
```

Comment:
I prefer leave the users report it by themself, that'll certainly make more sens to you.